### PR TITLE
Update of library

### LINKED ISSUE #2: Update this awesome package
Because of the Material UI new version, this package needs to have some updates.

### PR DESCRIPTION
**Pull request by amerllica**
This PR resolves #2

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,21 +23,17 @@
...
   },
   "homepage": "https://github.com/mojilla/material-snackbar-supplier#readme",
-  "peerDependencies": {
-    "@material-ui/core": "^3.0.1",
-    "@material-ui/icons": "^3.0.1",
-    "react": "^16.4.2"
+  "dependencies": {
+    "@material-ui/core": "^4.2.0",
+    "@material-ui/icons": "^4.2.1",
+    "react": "^16.8.6"
```
Comment (themojilla) [inline package.json:29]: Don't you think peerDependencies has benefited over dependencies in this case?
Anyone who attends to install it, he already installed material, react

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2019 Amer Lotfi Orimi
```
Comment (themojilla) [inline LICENSE]: Amer Lotfi Orimi ? :dancer:

Comment (amerllica) [reply LICENSE]: This is my mistake. I'll fix this.

File: package.json (hunk 1 above)
Comment (amerllica) [reply package.json:29]: Ok, got it. I do not know about `peerDependencies` and its benefits. Thanks, I'll fix this.

**Comment by amerllica** [comment]
I change the license file and the package.json file. thanks for your review.

**Comment by themojilla** [comment]
fixed #2

**Pull request merged**